src/vhdl/vhdl-sem_lib.adb

```diff
@@ -128,6 +129,21 @@ package body Vhdl.Sem_Lib is
 
       Vhdl.Canon.Canonicalize (Unit);
 
+      -- Synthesis checks
+      --------------------
+
+      -- Note: Calling Synthesis checks after Canon is more computationally
+      --       demanding than before. This is because, concurent assignments
+      --       has been converted to processes and are therefore checked too!
+      --       We stick with this solution because Canon pass fixes possibly
+      --       broken chain of concurent statements for us!
```

Comment:
It is not clear to me what the broken chains are.

Reply:
I thought that link lists of sequential statements could be "broken", having no links between individual statements  (I saw a coment somewhere in the code from which I understood that canon pass would handle this situation). Clearly, I got it wrong and no such thing could occur.